ivc/tests/add.rs

```diff
@@ -365,15 +365,13 @@ pub fn heavy_test_simple_add() {
         assert_eq!(s.len(), domain_size);
     });
 
-    let ivc_fixed_selectors_evals_d1: Vec<Evaluations<Fp, R2D<Fp>>> = ivc_fixed_selectors
-        .clone()
+    let ivc_fixed_selectors_evals_d1: Vec<Evaluations<Fp, R2D<Fp>>> = (&ivc_fixed_selectors)
         .into_par_iter()
-        .map(|w| Evaluations::from_vec_and_domain(w, domain.d1))
+        .map(|w| Evaluations::from_vec_and_domain(w.to_vec(), domain.d1))
```

Comment:
I wish we could create this from any slice (`from_slice_and_domain`?), so we could avoid converting slices to arrays...